modules/acquia_cms_common/config/optional/gdpr_fields.gdpr_fields_config.node.yml

```diff
@@ -0,0 +1,110 @@
+langcode: en
```

Comment:
Please move this file to ACMS Person module

Reply:
Done